### DISCUSSION TITLE
Will there ever be regular documentation?

### DISCUSSION THREAD

**Discussion by SeryiBaran** [unocss/unocss#2421] [Ideas]
Hello!

Will there be any plans to create the regular documentation (like WindiCSS) in addition to the interactive guide?

**Comment by zyyv** [comment]
We are building it in `docs-next` branch. If you want to help us to build it, you can make PRs to `docs-next` branch. And now, you can preview it early on [preview.unocss.dev](https://preview.unocss.dev)

**Comment by SeryiBaran** [comment]
Wow, thank you!
